src/main/java/frc/robot/commands/AlgaeOuttake.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.end_effector.EndEffector;
+public class AlgaeOuttake extends Command{
+    private EndEffector endEffector;
+    private DoubleSupplier algaeouttakespeedSupplier;
+    public AlgaeOuttake(EndEffector endEffector, DoubleSupplier algaeouttakespeed) {
+        this.endEffector=endEffector;
+        this.addRequirements(endEffector);
+    }
+
+public AlgaeOuttake(double algaeouttakespeed) {
+        //TODO Auto-generated constructor stub
+    }
+
+@Override
+public void initialize () 
```

Comment:
The end effector should be run not only at `initialize()` but rather the entire time the command is run in `execute()`.